lists/na/na-namra.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": {
+    "en": "Namibia Revenue Agency ",
+    "local": "English"
+  },
+  "url": "https://www.namra.org.na/",
+  "description": {
+    "en": "The Namibia Revenue Agency maintains the details of all businesses, non-profits and individuals who are required to submit income tax returns. All legal entities must be registered with the Revenue Agency upon incorporation. Information regarding an entity can be requested from the Revenue Agency by the registered representative of the entity, accountants or auditors.\n"
+  },
+  "coverage": [
+    "NA"
+  ],
+  "subnationalCoverage": [],
+  "structure": [],
```

Comment:
Please supply the type of entities covered by this list. As a tax authority, it is likely to cover companies, charities (ngos) and a large range of entities.